src/mame/heathkit/intr_cntrl.cpp

```diff
@@ -171,6 +176,15 @@ void z37_intr_cntrl::device_start()
 	m_fd_irq_raised = false;
 }
 
+void z37_intr_cntrl::device_reset()
+{
+	heath_intr_cntrl::device_reset();
+
+	m_interrupts_blocked = false;
+	m_drq_raised = false;
+	m_fd_irq_raised = false;
```

Comment:
You shouldn’t be changing inputs on reset.  If the devices connected to those inputs deassert them on reset, it’s their responsibility to push the change.